zone-nyan.el

```diff
@@ -743,13 +743,18 @@ It fires every 100ms.")
 (defvar zone-nyan-progress 0
   "Holds the current progress of the timer.")
 
+(defun zone-nyan-report-progress ()
+  "Report current nyan progress."
+
```

Comment:
Out of habit, didn’t check the other functions for coding style; sorry about that. Fixed.